### PR TITLE
Patch/include rubocop defaults

### PR DESCRIPTION
**Pull request by ktopolski**

### CODE REVIEW

File: config/base.yml [hunk 1]
```diff
@@ -1065,19 +1168,23 @@ Style/SelfAssignment:
 
 Style/Semicolon:
   Enabled: true
+  AllowAsExpressionSeparator: false
 
 Style/SingleLineMethods:
   Enabled: true
+  AllowIfMethodIsEmpty: false
```
Comment (ktopolski) [inline config/base.yml:1175]: @onebree this one is default to `true`, but to keep it consistent with standard's `Style/EmptyMethod`(expanded) I changed it to `false`

**Comment by ktopolski** [comment]
Hello Hunter,

There goes second part of rubocop defaults. If merging PR does not sound good enough to you, I can rebase with your changes and push to your repo once granted maintainer :)

Cheers!

**Comment by onebree** [comment]
@ktopolski I'll review this tomorrow. But thanks for the help! I was thinking of having a single commit to post to Standard, but add you as Co-Authored-By, so we both get credit (and they only need to merge a single commit).

**Comment by ktopolski** [comment]
Sure @onebree , I have no issue with merging to one commit and co-authoring :)

**Comment by onebree** [comment]
@ktopolski Just getting to this now, sorry for the delay! Thank you for your help!

**Pull request merged**